QgisModelBaker/utils/gui_utils.py

```diff
@@ -3,7 +3,7 @@
 import pathlib
 import re
 import warnings
-import xml.etree.cElementTree as CET
+import xml.etree.ElementTree as CET
```

Comment:
Only thing I wonder is why we imported here something as CET (but it's not the cElementTree but the ElementTree but I guess it's a typo and since those libraries behave the same (maybe the speed is different) we did not notice..

Reply:
Here is some context
https://github.com/opengisch/QgisModelBaker/issues/529#issuecomment-937519862

Reply:
And according to https://stackoverflow.com/a/50509403/2319028 the two are identical nowadays

Reply:
I remembered my conclusion in this test with German. But fine - when they are identical nowadays, then I trust pyupgrade, that the cElementTree is what we should use.